src/appleseed-max-impl/appleseedobjpropsmod/appleseedobjpropsmod.cpp

```diff
@@ -352,6 +356,14 @@ std::string AppleseedObjPropsMod::get_sss_set(const TimeValue t) const
     return str_value != nullptr ? wide_to_utf8(str_value) : std::string();
 }
 
+int AppleseedObjPropsMod::get_medium_priority(const TimeValue t) const
+{
+   int int_value = 0;
+   int_value = m_pblock->GetInt(paramIdNestedmediumPriority, t, FOREVER);
+   return int_value;
+}
+
```

Comment:
Our rules for vertical spacing are:

- Always use a *single* blank line to separate things.
- *Except* before block comments (such as the one below) which should be preceded by *two* blank lines.

Reply:
Ok